sdk/template/AzureTemplate/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 # Release History
+## 1.1.0
```

Comment:
This doesn't seem to be using the common changelog logic

Reply:
Fixed this up as well in the latest PR.